test/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildALibrary.cs

```diff
@@ -577,5 +577,15 @@ public void It_passes_ridless_target_to_compiler()
                 .Should()
                 .Pass();
         }
+
+        [Fact]
+        public void It_restores_only_ridless_tfm()
```

Comment:
> I think you'll get coverage from the full msbuild runs in Jenkins.

Yes, if this isn't failing in the full MSBuild Jenkins run, then I think it's not testing what we think it is (since the corresponding change to the NuGet targets isn't on the Jevkins machine).  I'd like us to understand why this is the case.

You can run tests against full MSBuild locally with `build -FullMSBuild`.